package.json

```diff
@@ -6,12 +6,12 @@
   "scripts": {
     "transpile": "tsc -p ./src/tsconfig.json",
     "pretest": "npm run transpile",
-    "test": "node -r esm ./test/index.js",
+    "test": "node -r esm_prateek ./test/index.js",
```

Comment:
What's `esm_prateek`? A fork that fixes an issue?

Reply:
Oh yes, the fix is in esm as well, I pinged jdd he is supposed to release the new version in this week probably.

I should add a to-do here